Bricks/net/api/response.h

```diff
@@ -0,0 +1,188 @@
+/*******************************************************************************
+The MIT License (MIT)
+
+Copyright (c) 2015 Dmitry "Dima" Korolev, <dmitry.korolev@gmail.com>.
+
+Permission is hereby granted, free of charge, to any person obtaining a copy
+of this software and associated documentation files (the "Software"), to deal
+in the Software without restriction, including without limitation the rights
+to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+copies of the Software, and to permit persons to whom the Software is
+furnished to do so, subject to the following conditions:
+
+The above copyright notice and this permission notice shall be included in all
+copies or substantial portions of the Software.
+
+THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
+SOFTWARE.
+*******************************************************************************/
+
+#ifndef BRICKS_NET_API_RESPONSE_H
+#define BRICKS_NET_API_RESPONSE_H
+
+// A convenience wrapper to pass in to `Request::operator()`, for the cases where
+// the best design is to return a value from one function to have the next one in the chain,
+// which can be `std::move(request)`, to pick it up.
+
+#include "request.h"
+
+#include "../exceptions.h"
+
+#include "../http/http.h"
+#include "../../strings/is_string_type.h"
+#include "../../cerealize/cerealize.h"
+
+namespace bricks {
+namespace net {
+namespace api {
+
+static_assert(strings::is_string_type<const char*>::value, "");
+static_assert(!strings::is_string_type<int>::value, "");
```

Comment:
They are already there.

And here they are only because I was testing something. Removing.